pkg/ami/api.go

```diff
@@ -60,8 +66,9 @@ func IsAvailable(api ec2iface.EC2API, id string) (bool, error) {
 // FindImage will get the AMI to use for the EKS nodes by querying AWS EC2 API.
 // It will only look for images with a status of available and it will pick the
 // image with the newest creation date.
-func FindImage(api ec2iface.EC2API, namePattern string) (string, error) {
+func FindImage(api ec2iface.EC2API, namePattern string, imageFamily string) (string, error) {
```

Comment:
I think map lookup should be all in the same place, let's just make it like this here:
```suggestion
func FindImage(api ec2iface.EC2API, namePattern, accountID string) (string, error) {
```